auth/src/main/java/com/firebase/ui/auth/ui/FragmentHelper.java

```diff
@@ -6,6 +6,7 @@
 import android.os.Bundle;
 import android.support.annotation.RestrictTo;
 import android.support.v4.app.Fragment;
+import android.support.v7.view.ContextThemeWrapper;
```

Comment:
I think we need to use android.view.ContextThemeWrapper (https://developer.android.com/reference/android/view/ContextThemeWrapper.html) instead of the support library version.  The framework version should not be restricted, and was added in API Level 1.